frontend/src/components/regions/CardDetails/utils.js

```diff
@@ -1,4 +1,5 @@
-function hasPermission({ authUser, user }) {
+
+const hasPermission = ({ authUser, user }) => {
```

Comment:
Can you please make this name more explicit? 
We are checking if a person has permission to manage cards.